.github/CODEOWNERS

```
@@ -6,30 +6,37 @@
 ################
 
 # Git Hub integration and bot rules
-/.github/                @AlexGhiondea @jsquire
+/.github/                @AlexGhiondea @jsquire @rickwinter
 
 ###########
 # SDK
 ###########
 
 # Catch all for non-code project files and unowned files | folders
-*                        @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama
-/sdk/                    @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama
+*                        @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @larryo
```

I am assuming this is supposed to be, @LarryOsterman 
```suggestion
*                        @rickwinter @ahsonkhan @antkmsft @vhvb1989 @gearama @LarryOsterman
```